CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
   as source of data in the client
 - perf: use perfect hash function in calculate_l1_gas_by_vm_usage
 - chore: add tests for tx hashing
+- split `primitives` crates into multiple smaller crates
```

Comment:
Why not add the changes on the top? When there is a conflict, the newest changes stay on directly

Reply:
oooooh. You are so right!!!